terra/reports.py

```diff
@@ -704,22 +736,46 @@ def get_individual_data_employee(employee_ids, start_date=None, end_date=None):
                 Value(0),
             ),
         )
-        .values("id", "total_requested", "total_spent", "days_away")
+        .values(
+            "id",
+            "profdev_spent",
+            "profdev_days_away",
+            "total_requested",
+            "total_spent",
+            "days_away",
+        )
     )
     return rows
 
 
 def employee_total_report(employee_ids, start_date, end_date):
     employee_totals = {}
     rows = get_individual_data_employee(employee_ids, start_date, end_date)
+    profdev_spending_cap = 3500
+    profdev_days_cap = 15
```

Comment:
@aprigge I think these "magic numbers" are spreading.  For example, these also appear in `terra/templatetags/terra_extras.py`.  Can they be pulled into a central location and defined as constants, then referenced as needed wherever relevant?  Maybe they can fit into a model; possibly a context, maybe something else?

If the spending cap goes up to $3800 next year, for example, we don't want to hunt down and change every use of 3500 in the project.

Which raises the question: as these caps change over time, will previous FY reports be affected?  I don't think so, other than highlighting / color coding, but something to consider.

If there's no immediate obvious best way to handle this centralization, I'm OK merging this now, as long as a ticket's created to address the resulting technical debt.

Thanks --Andy

Reply:
@akohler, good point. Let me centralize the caps before merging. But yes, in general we haven't figured out what to do when/if caps change. Thanks!